backend/cli/delete_day.py

```diff
@@ -14,4 +14,5 @@ def delete_day(params):
         cur.execute("DELETE" + QUERY + "is null", [params_start, params_end])
         cur.execute("SELECT start_interval" + QUERY + "NOT null", [params_start, params_end])
         for result in cur:
-            print("""Can't delete booked interval: {}""".format(result))
+            result = result[0]
```

Comment:
You may notice that you had to make the same changes in two similar functions. That may be a sign that the code is repetitive and refactoring may be a good option.